### ISSUE TITLE
Expectations in after and afterJSON

### ISSUE THREAD

**Issue by PyvesB** [IcedFrisby/IcedFrisby#206]
Hello,

The Readme claims the following:
```
.afterJSON(user => {
  // You can use any normal assertions here
  expect(1 + 1).to.equal(2)
})
```

However, this doesn't seem to be the case. I wrote this simple test:
```
const { expect } = require('chai')
const frisby =  require('icedfrisby')

frisby
  .create('example test')
  .get('https://google.com')
  .expectStatus(200)
  .afterJSON(json => {
    expect(1 + 1).to.equal(999999)
  })
  .after(() => {
    expect(true).to.be.false
  })
  .toss()
```

The test passes, neither of the expectations in the callbacks fail. Am I missing something here?

Thanks in advance.

Cheers,

Pyves

**Comment by paulmelnikow** [comment]
Hey Pyves! Thanks for the ping on this!

How are you running this? I put it into `frisby-test.js` and ran `./node_modules/.bin/mocha frisby-test.js` after installing icedfrisby and mocha. The test fails: "Error: Error parsing JSON string: Unexpected token < in JSON at position 0".

**Comment by PyvesB** [comment]
I'm running it with the same command, but I did check out the project and ran it on the current _master_ branch (and therefore the require is a relative path, `require('./lib/icedfrisby')`). However, I'm also facing similar issues with release `2.0.0-alpha.4`: I cannot get [these Shields tests](https://github.com/badges/shields/blob/master/services/node/node.tester.js) to fail, no matter how I change the `expectSemverRange` checker.

**Comment by paulmelnikow** [comment]
Released in **2.0.0-alpha.5**.
